packages/ui/src/redux/store.ts

```diff
@@ -4,17 +4,25 @@ import storage from 'redux-persist/lib/storage';
 import accountsSlice from 'redux/slices/accounts';
 import appSlice from 'redux/slices/app';
 import setupWalletSlice from 'redux/slices/setup-wallet';
+import settingSlice from './slices/settings';
```

Comment:
Fix this import path to import from source root folder, please check other places as well.